requirements-dev.txt

```diff
@@ -3,4 +3,5 @@ pycodestyle
 pylint>=2.3,<2.4
 pylintfileheader>=0.0.2
 vcrpy
-pproxy==1.2.2
+pproxy==1.2.2; python_version > '2.6' and python_version <= '3.5'
```

Comment:
Actually, I preferred the previous iteration - if somebody is attempting to use Python 2, they will have bigger problems trying to run the package than the version of a development dependency :) Can you remove the bit about `2.6`, for simplicity?